pinot-servicemanager/etc/pinot-broker.conf

```diff
@@ -1,6 +0,0 @@
-pinot.broker.client.queryPort=8099
```

Comment:
are all default?

Reply:
Ah! see, it is copied via install

Reply:
not sure.. just transferred over from other files